### PR TITLE
Update react query

### PR DESCRIPTION
**Pull request by elkebab**
# Description

Update `react-query` to new major version (v5).
Ganske mange endringer, så litt vrient å sjekke alt så nøye: https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5
Tenker det er like greit å sjekke via e2e-tester etter merging til dev.

# Checklist

Ensure that the changes are aligned with the expectations of PO/designer before marking the PR as ready for review.

Also ensure that the following criterias are met:

- [ ] The changes are working as expected
- [x] The changes are tested OK for different screen sizes
- [x] The changes are tested OK for a11y
- [x] Interactive elements have data-testids
- [x] I have done a QA of my own changes

_Note: some of these criterias may not always be relevant, and can simply be marked as completed._

### CODE REVIEW

**Review by joachimjorgensen — APPROVED** [review]
Godt jobba med grovarbeid 💪 Enig i at e2e gir en god pekepinn på at alt er greit

**Comment by elkebab** [comment]
@SocketSecurity ignore npm/core-js@3.29.1
    @SocketSecurity ignore npm/core-js-pure@3.31.0
    @SocketSecurity ignore npm/cypress@13.8.1

File: src/api/searchApi.ts [hunk 1]
```diff
@@ -446,7 +446,7 @@ export const fetchResults = async (params: FetchResultsParams, signal?: AbortSig
   searchParams.set(ResultParam.Sort, params.sort ?? 'desc');
 
   const getResults = await apiRequest2<SearchResponse2<Registration, RegistrationAggregations>>({
-    url: `${SearchApiPath.Registrations}?${searchParams.toString()}`,
+    url: `${SearchApiPath.Registrations}?${searchParams.toString()}&loff=32`,
```
Comment (elkebab) [inline src/api/searchApi.ts:449]:
```suggestion
    url: `${SearchApiPath.Registrations}?${searchParams.toString()}`,
```

**Review by joachimjorgensen — DISMISSED** [review]
LGTM

**Pull request merged**